src/components/_video/VideoTile/VideoTile.tsx

```diff
@@ -14,14 +14,18 @@ export type VideoTileProps = {
 } & VideoTileBaseMetaProps &
   Pick<VideoTileBaseProps, 'progress' | 'className'>
 
-export const VideoTile: React.FC<VideoTileProps> = ({ id, onNotFound, ...metaProps }) => {
+export const VideoTile: React.FC<VideoTileProps> = React.memo(({ id, onNotFound, ...metaProps }) => {
   const { video, loading, videoHref, thumbnailPhotoUrl, avatarPhotoUrl, isLoadingThumbnail, isLoadingAvatar } =
     useVideoSharedLogic({
       id,
       isDraft: false,
       onNotFound,
     })
 
+  const onCopyVideoURLClick = useCallback(() => {
```

Comment:
Small detail but according to our styleguide (the mental one, not sure if we have this documented), this should be called `handleCopyVideoURLClick`

Reply:
btw, we could collect our mental styleguides one day and make some kind of document out of it 😉. I think this would be useful for new devs.

Reply:
Yes that's for sure! We actually have an old (a bit outdated at this point) piece of styleguide [here](https://github.com/Joystream/atlas/blob/master/docs/styleguide.md). I'd love to get this updated but there just always are some other, more important things to do :(